source/elements/oneDNN/source/replacements.inc.rst

```diff
@@ -325,9 +325,9 @@
 .. |memory::get_data_handle| replace:: :any:`dnnl::memory::get_data_handle`
 .. |memory::get_desc| replace:: :any:`dnnl::memory::get_desc`
 .. |memory::get_engine| replace:: :any:`dnnl::memory::get_engine`
-.. |memory::get_sycl_buffer| replace:: :any:`dnnl::memory::get_sycl_buffer`
+.. |memory::get_buffer| replace:: :any:`dnnl::memory::get_buffer`
+.. |memory::set_buffer| replace:: :any:`dnnl::memory::set_buffer`
```

Comment:
we use set_data_handle in the api. probably should indeed change to `set_buffer()` (see dnnl_sycl.hpp`)

Reply:
Updated `set_data_handle` to `set_buffer` in `dnnl_sycl.hpp`